server/node.go

```diff
@@ -320,6 +324,7 @@ func (n *Node) bootstrapStores(bootstraps *list.List) {
 // node's address is gossiped with node ID as the gossip key.
 func (n *Node) connectGossip() {
 	log.Infof("connecting to gossip network to verify cluster ID...")
+	// TODO(DT): Should this receive with a timeout and node stop in a select statement?
```

Comment:
The gossip object will print out various log statements regarding its attempts to connect which will be sufficient to clue the operator in to the fact that the connection is having troubles, should that be the case. I don't think you'd ever want a timeout here. Go ahead and change this TODO into a comment to that effect.